src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -238,29 +276,22 @@ public void Execute<TResult>(RavenCommand<TResult> command, JsonOperationContext
                     else
                     {
                         response = await command.SendAsync(client, request, token).ConfigureAwait(false);
+                        if (response.Headers.Contains(Constants.Headers.ForceTopologyUpdate))
```

Comment:
And why are we forcing the client to wait for the next topology when we already have the response for what it asked for?